src/tools/cgexec.c

```diff
@@ -171,12 +188,128 @@ int main(int argc, char *argv[])
 		}
 	}
 
-	/* Now exec the new process */
-	ret = execvp(argv[optind], &argv[optind]);
-	if (ret == -1) {
-		err("%s", strerror(errno));
+	if (!replace_idle) {
+		/* Now exec the new process */
+		ret = execvp(argv[optind], &argv[optind]);
+		err("exec failed:%s", strerror(errno));
```

Comment:
nitpick - looks like the `if (ret == -1)` was removed, and now this error message will always be printed.